_studio/shared/mfx_trace/CMakeLists.txt

```diff
@@ -28,5 +28,5 @@ file( GLOB_RECURSE srcs "${MSDK_STUDIO_ROOT}/shared/mfx_trace/src/*.cpp" )
 list( APPEND sources ${srcs})
 
 set( defs "" )
-make_library( mfx_trace none static )
+make_library( mfx_trace none static "nosafestring")
```

Comment:
Don't do this, please. I guess this don't affect your change. And we in any case still have a dependency from this library for the rest of mediasdk library.

Reply:
I thought this will affect only mfx_trace target, which no longer needs SafeStrings library, isn't it?

Reply:
it will, still not needed.